[2023-12-28T16:09:06.890+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-17T04:00:00+00:00/task_id=Create_Table permission to 509
[2023-12-28T16:09:06.908+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-17T04:00:00+00:00/task_id=Create_Table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-17T04:00:00+00:00/task_id=Create_Table'
[2023-12-28T16:09:07.061+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-17T04:00:00+00:00/task_id=Create_Table permission to 509
[2023-12-28T16:09:07.065+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-17T04:00:00+00:00/task_id=Create_Table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-17T04:00:00+00:00/task_id=Create_Table'
[2023-12-28T16:09:07.347+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: air_quality.Create_Table scheduled__2023-12-17T04:00:00+00:00 [queued]>
[2023-12-28T16:09:07.416+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: air_quality.Create_Table scheduled__2023-12-17T04:00:00+00:00 [queued]>
[2023-12-28T16:09:07.419+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-12-28T16:09:07.681+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): Create_Table> on 2023-12-17 04:00:00+00:00
[2023-12-28T16:09:07.691+0000] {standard_task_runner.py:57} INFO - Started process 559 to run task
[2023-12-28T16:09:07.716+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'air_quality', 'Create_Table', 'scheduled__2023-12-17T04:00:00+00:00', '--job-id', '765', '--raw', '--subdir', 'DAGS_FOLDER/airQuality.py', '--cfg-path', '/tmp/tmp1nuaa5dy']
[2023-12-28T16:09:07.723+0000] {standard_task_runner.py:85} INFO - Job 765: Subtask Create_Table
[2023-12-28T16:09:07.968+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-17T04:00:00+00:00/task_id=Create_Table permission to 509
[2023-12-28T16:09:07.971+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-17T04:00:00+00:00/task_id=Create_Table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-17T04:00:00+00:00/task_id=Create_Table'
[2023-12-28T16:09:07.978+0000] {task_command.py:416} INFO - Running <TaskInstance: air_quality.Create_Table scheduled__2023-12-17T04:00:00+00:00 [running]> on host d67dd4682ccf
[2023-12-28T16:09:08.406+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='ghifari' AIRFLOW_CTX_DAG_ID='air_quality' AIRFLOW_CTX_TASK_ID='Create_Table' AIRFLOW_CTX_EXECUTION_DATE='2023-12-17T04:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-12-17T04:00:00+00:00'
[2023-12-28T16:09:08.449+0000] {base.py:73} INFO - Using connection ID 'neonServerDB-Postgres' for task execution.
[2023-12-28T16:09:12.829+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 192, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 209, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/airQuality.py", line 133, in createTableSQL
    cursor.execute(createQuery)
psycopg2.errors.UniqueViolation: duplicate key value violates unique constraint "pg_class_relname_nsp_index"
DETAIL:  Key (relname, relnamespace)=(air_quality_id_seq, 2200) already exists.

[2023-12-28T16:09:12.860+0000] {taskinstance.py:1400} INFO - Marking task as UP_FOR_RETRY. dag_id=air_quality, task_id=Create_Table, execution_date=20231217T040000, start_date=20231228T160907, end_date=20231228T160912
[2023-12-28T16:09:12.891+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 765 for task Create_Table (duplicate key value violates unique constraint "pg_class_relname_nsp_index"
DETAIL:  Key (relname, relnamespace)=(air_quality_id_seq, 2200) already exists.
; 559)
[2023-12-28T16:09:12.925+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-12-28T16:09:12.989+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-12-28T16:52:07.290+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-17T04:00:00+00:00/task_id=Create_Table permission to 509
[2023-12-28T16:52:07.297+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-17T04:00:00+00:00/task_id=Create_Table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-17T04:00:00+00:00/task_id=Create_Table'
[2023-12-28T16:52:07.473+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-17T04:00:00+00:00/task_id=Create_Table permission to 509
[2023-12-28T16:52:07.475+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-17T04:00:00+00:00/task_id=Create_Table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-17T04:00:00+00:00/task_id=Create_Table'
[2023-12-28T16:52:07.664+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: air_quality.Create_Table scheduled__2023-12-17T04:00:00+00:00 [queued]>
[2023-12-28T16:52:07.747+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: air_quality.Create_Table scheduled__2023-12-17T04:00:00+00:00 [queued]>
[2023-12-28T16:52:07.749+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-12-28T16:52:07.874+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): Create_Table> on 2023-12-17 04:00:00+00:00
[2023-12-28T16:52:07.897+0000] {standard_task_runner.py:57} INFO - Started process 329 to run task
[2023-12-28T16:52:07.925+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'air_quality', 'Create_Table', 'scheduled__2023-12-17T04:00:00+00:00', '--job-id', '905', '--raw', '--subdir', 'DAGS_FOLDER/airQuality.py', '--cfg-path', '/tmp/tmplgptjv4o']
[2023-12-28T16:52:07.930+0000] {standard_task_runner.py:85} INFO - Job 905: Subtask Create_Table
[2023-12-28T16:52:08.207+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-17T04:00:00+00:00/task_id=Create_Table permission to 509
[2023-12-28T16:52:08.208+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-17T04:00:00+00:00/task_id=Create_Table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-17T04:00:00+00:00/task_id=Create_Table'
[2023-12-28T16:52:08.227+0000] {task_command.py:416} INFO - Running <TaskInstance: air_quality.Create_Table scheduled__2023-12-17T04:00:00+00:00 [running]> on host d67dd4682ccf
[2023-12-28T16:52:08.971+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='ghifari' AIRFLOW_CTX_DAG_ID='air_quality' AIRFLOW_CTX_TASK_ID='Create_Table' AIRFLOW_CTX_EXECUTION_DATE='2023-12-17T04:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-12-17T04:00:00+00:00'
[2023-12-28T16:52:09.020+0000] {base.py:73} INFO - Using connection ID 'neonServerDB-Postgres' for task execution.
[2023-12-28T16:52:09.622+0000] {airQuality.py:139} INFO - Table air_quality Already Exist
[2023-12-28T16:52:09.687+0000] {python.py:194} INFO - Done. Returned value was: 
        CREATE TABLE air_quality (
            id serial primary key, 
            city varchar(250),
            country varchar(250), 
            unit varchar(20), 
            last_value float8, 
            lat float8,
            lng float8
        )
    
[2023-12-28T16:52:09.809+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=air_quality, task_id=Create_Table, execution_date=20231217T040000, start_date=20231228T165207, end_date=20231228T165209
[2023-12-28T16:52:09.978+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-12-28T16:52:10.059+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-12-28T16:55:10.483+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-17T04:00:00+00:00/task_id=Create_Table permission to 509
[2023-12-28T16:55:10.502+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-17T04:00:00+00:00/task_id=Create_Table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-17T04:00:00+00:00/task_id=Create_Table'
[2023-12-28T16:55:10.750+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-17T04:00:00+00:00/task_id=Create_Table permission to 509
[2023-12-28T16:55:10.757+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-17T04:00:00+00:00/task_id=Create_Table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-17T04:00:00+00:00/task_id=Create_Table'
[2023-12-28T16:55:10.957+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: air_quality.Create_Table scheduled__2023-12-17T04:00:00+00:00 [queued]>
[2023-12-28T16:55:10.992+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: air_quality.Create_Table scheduled__2023-12-17T04:00:00+00:00 [queued]>
[2023-12-28T16:55:10.993+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-12-28T16:55:11.065+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): Create_Table> on 2023-12-17 04:00:00+00:00
[2023-12-28T16:55:11.074+0000] {standard_task_runner.py:57} INFO - Started process 1314 to run task
[2023-12-28T16:55:11.109+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'air_quality', 'Create_Table', 'scheduled__2023-12-17T04:00:00+00:00', '--job-id', '1152', '--raw', '--subdir', 'DAGS_FOLDER/airQuality.py', '--cfg-path', '/tmp/tmp21qvtvh9']
[2023-12-28T16:55:11.114+0000] {standard_task_runner.py:85} INFO - Job 1152: Subtask Create_Table
[2023-12-28T16:55:11.359+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-17T04:00:00+00:00/task_id=Create_Table permission to 509
[2023-12-28T16:55:11.360+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-17T04:00:00+00:00/task_id=Create_Table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-17T04:00:00+00:00/task_id=Create_Table'
[2023-12-28T16:55:11.366+0000] {task_command.py:416} INFO - Running <TaskInstance: air_quality.Create_Table scheduled__2023-12-17T04:00:00+00:00 [running]> on host d67dd4682ccf
[2023-12-28T16:55:11.842+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='ghifari' AIRFLOW_CTX_DAG_ID='air_quality' AIRFLOW_CTX_TASK_ID='Create_Table' AIRFLOW_CTX_EXECUTION_DATE='2023-12-17T04:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-12-17T04:00:00+00:00'
[2023-12-28T16:55:11.905+0000] {base.py:73} INFO - Using connection ID 'neonServerDB-Postgres' for task execution.
[2023-12-28T16:55:12.321+0000] {airQuality.py:139} INFO - Table air_quality Already Exist
[2023-12-28T16:55:12.405+0000] {python.py:194} INFO - Done. Returned value was: 
        CREATE TABLE air_quality (
            id serial primary key, 
            city varchar(250),
            country varchar(250), 
            unit varchar(20), 
            last_value float8, 
            lat float8,
            lng float8
        )
    
[2023-12-28T16:55:12.527+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=air_quality, task_id=Create_Table, execution_date=20231217T040000, start_date=20231228T165510, end_date=20231228T165512
[2023-12-28T16:55:12.942+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-12-28T16:55:13.032+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
